generative/3d_ldm/README.md

```diff
@@ -0,0 +1,137 @@
+# 3D Latent Diffusion Example
+This folder contains an example for training and validating a 3D Latent Diffusion Model on Brats data. The example includes support for multi-GPU training with distributed data parallelism based on a [tutorial designed for using single GPU](https://github.com/Project-MONAI/GenerativeModels/blob/main/tutorials/generative/3d_ldm/3d_ldm_tutorial.ipynb).
+
+The workflow of the Latent Diffusion Model is depicted in the figure below. It begins by training an autoencoder in pixel space to encode images into latent features. Following that, it trains a diffusion model in the latent space to denoise the noisy latent features. During inference, it first generates latent features from random noise by applying multiple denoising steps using the trained diffusion model. Finally, it decodes the denoised latent features into images using the trained autoencoder.
+<p align="center">
+  <img src="./figs/ldm.png" alt="latent diffusion scheme")
+</p>
+
+MONAI latent diffusion model implementation is based on the following papers:
+
+[**Latent Diffusion:** Rombach, Robin, et al. "High-resolution image synthesis with latent diffusion models." CVPR 2022.](https://openaccess.thecvf.com/content/CVPR2022/papers/Rombach_High-Resolution_Image_Synthesis_With_Latent_Diffusion_Models_CVPR_2022_paper.pdf)
+
+This network is designed as a demonstration to showcase the training process for this type of network using MONAI. To achieve optimal performance, it is recommended that users have a GPU with memory larger than 32G to accommodate larger networks and attention layers.
+
+### 1. Data
+
+The dataset we are experimenting with in this example is BraTS 2018 data (https://www.med.upenn.edu/sbia/brats2018.html).
+
+BraTS 2018 is a public dataset of brain MR images. Using these images, the goal is to generate images that look similar to the images in BraTS 2018 dataset.
+
+The data can be automatically downloaded at the beginning of training.
+
+Disclaimer: We are not the host of the data. Please make sure to read the requirements and usage policies of the data and give credit to the authors of the dataset! We acknowledge the National Cancer Institute and the Foundation for the National Institutes of Health, and their critical role in the creation of the free publicly available LIDC/IDRI Database used in this study.
+
+### 2 Installation
```

Comment:
Missing the `.` between 2 and Installation.

Alternatively, I would suggest to remove the number so that we don't have to keep an eye on these details in the future, just like `### Data`, ` ### Installation`, `### Run the example`, and etc